requirements.txt

```diff
@@ -3,3 +3,5 @@ geojson>=2.4.1
 pygeoj
 pyshp
 python-dateutil
+pandas
```

Comment:
Is it really required to add pandas? AFAICS that is a rather hefty tool if we only use date parsing..

Maybe you can document the different libraries you tried in the related issue?

Reply:
As we require to parse all dates in the files in the same format, we need to handle ambiguous dates (e.g 01.05.2010). We can do this by extracting the preferred guessed format of all individual dates. After that, we have a list of a few potential 'date time formats' that we can use to parse all the dates. Some formats are only going to work for some dates (ambiguous dates) but not for other dates. This allow us to discard time formats and find the right one (the one format that correctly parse all dates). For this reason we required a function that returns the 'guessed' format. Pandas .datetimes is the only tool that returns information about the guessed format of string in the parsing process.

These feature is not available in other  'lighter tools' as dateparser: https://github.com/scrapinghub/dateparser/issues/341